### ISSUE TITLE
AttributeError: 'BERTopic' object has no attribute 'topic_aspects_'

### ISSUE THREAD

**Issue by AbuBakarrr** [MaartenGr/BERTopic#1392]
After updating BERTopic to the latest version, I encountered an error whenever I tried running the **get_topic_info** command. In the previous version (0.13), this command worked without any issues. The specific error message I received was:

```
AttributeError: 'BERTopic' object has no attribute 'topic_aspects_'
```

![image](https://github.com/MaartenGr/BERTopic/assets/34186478/00752665-7599-48d8-98a8-5df10ca21611)

**Comment by MaartenGr** [comment]
Did you load in a previously saved model? If so, then it is important to keep the version as they are otherwise you might run into these API-related issues.

**Comment by keyeun** [comment]
I have the same issue although my BERtopic version is [Version 0.15.0].

**Comment by MaartenGr** [comment]
@keyeun Same question as the above, did you load in a previously saved model? If so, did you make sure you used the exact same BERTopic version?

**Comment by g3rfx** [comment]
I am facing the same issue. 
Is there a way to convert models which were saved using "pickle" and BERTopic 0.14.1 into the "safetensors" format added in version 0.15?

**Comment by MaartenGr** [comment]
@g3rfx No, that is not possible since the safetensors format was introduced in a later version that required internal changes to the API. You would have to re-train the model in v0.15 and save it with safetensors in order to use it in v0.15 and up.
